<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Triviador</class>
 <widget class="QDialog" name="Triviador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Triviador</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(166, 154, 128);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="joinButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(163, 130, 54);
	border: none;
	padding-top: 5px;
	color: rgb(31, 22, 1);
	border-radius:10px;
	border-left:  1px solid rgb(135, 106, 39);
	border-right: 1px solid rgb(135, 106, 39);
	border-bottom: 5px solid rgb(135, 106, 39);
	font: bold large &quot;Monospace&quot;;
	

}
QPushButton:hover {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-bottom: 5px solid rgb(115, 81, 5);
}
QPushButton:pressed {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-top: 5px solid rgb(115, 81, 5);
	padding-top: -5px;
	border-bottom: none;
}</string>
         </property>
         <property name="text">
          <string>Join</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="leaveButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(163, 130, 54);
	border: none;
	padding-top: 5px;
	color: rgb(31, 22, 1);
	border-radius:10px;
	border-left:  1px solid rgb(135, 106, 39);
	border-right: 1px solid rgb(135, 106, 39);
	border-bottom: 5px solid rgb(135, 106, 39);
	font: bold large &quot;Monospace&quot;;
	

}
QPushButton:hover {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-bottom: 5px solid rgb(115, 81, 5);
}
QPushButton:pressed {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-top: 5px solid rgb(115, 81, 5);
	padding-top: -5px;
	border-bottom: none;
}</string>
         </property>
         <property name="text">
          <string>Leave</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readyButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(163, 130, 54);
	border: none;
	padding-top: 5px;
	color: rgb(31, 22, 1);
	border-radius:10px;
	border-left:  1px solid rgb(135, 106, 39);
	border-right: 1px solid rgb(135, 106, 39);
	border-bottom: 5px solid rgb(135, 106, 39);
	font: bold large &quot;Monospace&quot;;
	

}
QPushButton:hover {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-bottom: 5px solid rgb(115, 81, 5);
}
QPushButton:pressed {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-top: 5px solid rgb(115, 81, 5);
	padding-top: -5px;
	border-bottom: none;
}</string>
         </property>
         <property name="text">
          <string>Ready</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="unreadyButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(163, 130, 54);
	border: none;
	padding-top: 5px;
	color: rgb(31, 22, 1);
	border-radius:10px;
	border-left:  1px solid rgb(135, 106, 39);
	border-right: 1px solid rgb(135, 106, 39);
	border-bottom: 5px solid rgb(135, 106, 39);
	font: bold large &quot;Monospace&quot;;
	

}
QPushButton:hover {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-bottom: 5px solid rgb(115, 81, 5);
}
QPushButton:pressed {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-top: 5px solid rgb(115, 81, 5);
	padding-top: -5px;
	border-bottom: none;
}</string>
         </property>
         <property name="text">
          <string>Unready</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>168</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>238</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(163, 130, 54);
	border: none;
	padding-top: 5px;
	color: rgb(31, 22, 1);
	border-radius:10px;
	border-left:  1px solid rgb(135, 106, 39);
	border-right: 1px solid rgb(135, 106, 39);
	border-bottom: 5px solid rgb(135, 106, 39);
	font: bold large &quot;Monospace&quot;;
	

}
QPushButton:hover {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-bottom: 5px solid rgb(115, 81, 5);
}
QPushButton:pressed {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-top: 5px solid rgb(115, 81, 5);
	padding-top: -5px;
	border-bottom: none;
}</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>238</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="message">
     <property name="styleSheet">
      <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">border: none;
	padding-top: 1px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Welcome !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Logged in as:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Games Played:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Wins:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">QLabel{
border: none;
border-radius:10px;
background-color: rgb(245, 232, 206);
}</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;(email)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">border: none;
border-radius:10px;
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;(nr)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">border: none;
border-radius:10px;
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;(nr)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>198</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true">	border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchUserButton">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(163, 130, 54);
	border: none;
	padding-top: 5px;
	color: rgb(31, 22, 1);
	border-radius:10px;
	border-left:  1px solid rgb(135, 106, 39);
	border-right: 1px solid rgb(135, 106, 39);
	border-bottom: 5px solid rgb(135, 106, 39);
	font: bold large &quot;Monospace&quot;;
	

}
QPushButton:hover {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-bottom: 5px solid rgb(115, 81, 5);
}
QPushButton:pressed {
	background-color: rgb(181, 132, 20);
	border-left:  1px solid rgb(115, 81, 5);
	border-right: 1px solid rgb(115, 81, 5);
	border-top: 5px solid rgb(115, 81, 5);
	padding-top: -5px;
	border-bottom: none;
}</string>
           </property>
           <property name="text">
            <string>Search User</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="styleSheet">
              <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;User:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Games Played:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Wins:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="styleSheet">
              <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Last Online:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="styleSheet">
              <string notr="true">border: none;
border-radius:10px;
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">border: none;
border-radius:10px;
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;nr&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="styleSheet">
              <string notr="true">border: none;
border-radius:10px;
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;nr&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="styleSheet">
              <string notr="true">border: none;
border-radius:10px;
background-color: rgb(245, 232, 206);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="userLobbyText">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
	padding-top: 5px;
	color: rgb(163, 130, 54);
	border-radius:10px;
	border-top:1px solid rgb(163, 130, 54);
	border-left:  1px solid rgb(163, 130, 54);
	border-right: 1px solid rgb(163, 130, 54);
	border-bottom: 1px solid rgb(163, 130, 54);
background-color: rgb(245, 232, 206);</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
